### ISSUE TITLE
JsonDecodingException: TranscodeReasons is not a string

### ISSUE THREAD

**Issue by GiviMAD** [jellyfin/jellyfin-sdk-kotlin#557]
Getting this error, using the latest Jellyfin version (10.8.5) and the latest sdk version (1.3.6).
Seems like it expects TranscodeReasons to be a String?
```
kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 5217: Expected beginning of the string, but got [ at path: $[0].TranscodingInfo.TranscodeReasons
JSON input: .....hannels":6,"TranscodeReasons":["ContainerBitrateExceedsLimit.....
	at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:24) ~[bundleFile:?]
	at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:32) ~[bundleFile:?]
	at kotlinx.serialization.json.internal.AbstractJsonLexer.fail(AbstractJsonLexer.kt:528) ~[bundleFile:?]
```

**Comment by GiviMAD** [comment]
Is it related to this?
https://github.com/jellyfin/jellyfin-sdk-kotlin/issues/510

**Comment by GiviMAD** [comment]
Thank you for the response.

> From what I could see this issue only occurred in WebSocket messages, in which case you can stop listening for session updates to prevent the exception.

I'm still not using the WebSocket. I almost sure I'm getting the error from this call:
```
new SessionApi(jellyApiClient).getSessions(...
```
So related to sessions but not with the WebSocket.

**Comment by nielsvanvelzen** [comment]
Ah right, the field is also available in the session list. I'll look at backporting the bugfix to 0.13.z later.

**Comment by nielsvanvelzen** [comment]
The latest release v1.3.7 fixes this issue with the patch from #542. Hopefully we'll see a proper fix in Jellyfin 10.9 so we can use an enum instead of string too.
